src/monitors.ts

```diff
@@ -0,0 +1,180 @@
+import { SERVERLESS_MONITORS } from "./serverless_monitors";
+import { updateMonitor, createMonitor, deleteMonitor, getExistingMonitors } from "./monitor-api-requests";
+
+export interface MonitorParams {
+  [key: string]: any;
```

Comment:
Can this interface really contain any key/value pair, or are there more specific rules for what is and is not a valid `MonitorParams`?

Reply:
The syntax is designed to be fairly flexible so that customers can choose which parameters to define and which ones they choose to leave out, so it's difficult to establish specific rules for what is valid and not. Ultimately, if the params are invalid, the API request will fail and it'll log an invalid syntax error message so I thought it would be best to just leave it as any general key/value pair